### ISSUE TITLE
make-gl-bitmap ignores get-legacy? property of gl-config.

### ISSUE THREAD

**Issue by elfprince13** [racket/gui#137]
For reasons that are not immediately clear, an OpenGL-enabled canvas respects the `get-legacy?` property on the `gl-config%` used to create it, whereas an OpenGL-enabled bitmap does not.

Compare this code, used in creating a canvas:
https://github.com/racket/gui/blob/84be70566c39188b1e08f2fe1a411831b596f8cb/gui-lib/mred/private/wx/cocoa/canvas.rkt#L218-L228

And this code, used in creating a bitmap:
https://github.com/racket/gui/blob/376ebd25dc7aab31843398dfacc5cae558dad7af/gui-lib/mred/private/wx/cocoa/cgl.rkt#L225-L249

Possibly relevant, the bitmap version uses the lower-level CGL API, rather than higher level NS API.

**Comment by elfprince13** [comment]
FWIW, the corresponding constants for `NSOpenGLProfileVersionLegacy` and `NSOpenGLProfileVersion3_2Core` appear to be named `kCGLOGLPVersion_Legacy` and `kCGLOGLPVersion_3_2_Core` (cf `CGLTypes.h`), and they have the same values (0x1000, and 0x3200, respectively).

The property key corresponding to `NSOpenGLPFAOpenGLProfile` for the CGL API appears to be called `kCGLPFAOpenGLProfile`, and also has an identical value.

**Comment by elfprince13** [comment]
Closed by #138.

For posterity, if anyone wants a simple test to determine if their `make-gl-bitmap` is behaving properly, this code should do the trick:

```racket
#lang racket

(require racket/gui)
(require opengl)
(require ffi/vector)

(define test-opengl-version
  (lambda (legacy?)

    (define my-config (new gl-config%))
    (send my-config set-legacy? legacy?)
    (define my-bitmap (make-gl-bitmap 256 256 my-config))

    (define my-bitmap-dc (new bitmap-dc% [bitmap
                                          my-bitmap]))

    (define my-gl-context (send my-bitmap-dc get-gl-context))
    (define version (send my-gl-context call-as-current
                          (lambda ()
                            (with-handlers ([exn:fail?
                                             (lambda (v) (glGetString GL_VERSION))])
                              (~a
                               (s32vector-ref (glGetIntegerv GL_MAJOR_VERSION) 0) "."
                               (s32vector-ref (glGetIntegerv GL_MINOR_VERSION) 0) "\n")))))
    version))

(display (test-opengl-version #f))

(display (test-opengl-version #t))
```


It could also be adapted fairly easily for a GL canvas, but I didn't implement that here.
